### DISCUSSION TITLE
Stand-alone app with pre-loaded tree

### DISCUSSION THREAD

**Discussion by hayoul** [PeWu/topola-viewer#150] [General]
Hello, thanks for sharing this on GH this is an awesome project.
I would like to present the genealogy tree page on my own server with my own GEDECOM file without giving the option to load any other files. I have extracted the JS files but can’t get to the search feature. 
maybe there is an easier way? Do you happen to have this option when creating the package? Or any guidance on  where I should look at?

**Comment by IstvanZachar** [comment]
This would be useful!

**Comment by PeWu** [comment]
If you want to host the Topola Viewer yourself, take a look at the [Build for your own data only](https://github.com/PeWu/topola-viewer/blob/master/README.md#build-for-your-own-data-only) section of the readme.

If you want to use the Topola Viewer version hosted on GitHub and hide the options that load other data, add `&embedded=true` to the URL once you load your GEDCOM file from a URL. You'll get a link like this:
https://pewu.github.io/topola-viewer/#/view?url=https%3A%2F%2Fchronoplexsoftware.com%2Fmyfamilytree%2Fsamples%2FThe%2520Kennedy%2520Family.gdz&embedded=true

**Reply by IstvanZachar** [reply]
Thanks PeWu for the quick answer, will try it out!
